Hera Myroslava/HW5/2.py

```diff
@@ -0,0 +1,7 @@
+n = int(input("Enter the value of n: "))
```

Comment:
I recommend using more meaningful variable names
look at this code
n = int(input('enter some number from fibonacci list= '))

fibonacci_numbers = [0, 1]

for i in range(2,n):
        fibonacci_numbers.append(fibonacci_numbers[i-1]+fibonacci_numbers[i-2])
print(f'there are some numbers from fibanacci till that you entered {n},\n {fibonacci_numbers}')